CHANGELOG.rst

```diff
@@ -4,6 +4,19 @@ CHANGE THIS IN changelogs/config.yaml! Release Notes
 
 .. contents:: Topics
 
+v3.1.0
+======
+
+Release Summary
+---------------
+
+This release adds a new role ``ec2_networking_resources`` and a pattern for creating an ec2 instance.
+
+Minor Changes
```

Comment:
antsibull-changelog should be able to detect new roles https://github.com/ansible-community/antsibull-changelog/blob/main/CHANGELOG.md#minor-changes-17.